robot/servo.py

```diff
@@ -114,13 +112,6 @@ def __init__(self, socket_path):
             for x in gpio_pins
         }
 
-    @property
-    def serial(self):
-        """Serial number of the board."""
-        return self._serial
-
-    # Servo code
```

Comment:
Unless they're wrong, I'd be in favour of keeping the comments which group the methods into "Servo" and "GPIO" chunks.